views/healthcheck/index.html

```diff
@@ -15,8 +15,10 @@
       <option v-repeat="options" value="{{$value}}" class="option-{{$index}}">{{ langmap($value) }}</option>
```

Comment:
Instead of adding another button, just add another element above here like this
```
<option value="use-default">Use device default</option>
```

and then in the `selectedLang` function, check if the `e.target.value === 'use-default'`